hieradata/common.yaml

```diff
@@ -0,0 +1,2 @@
+---
+influx_db_pwd: 'root'
```

Comment:
this should be named `statsd:: influxdb_password ` to make use of automatic data lookup